packages/csv-parser-price/src/main.js

```diff
@@ -183,9 +183,11 @@ export default class CsvParserPrice {
 // Easiest way to wrap the getCustomTypeDefinition in the memoize method
 CsvParserPrice.prototype.getCustomTypeDefinition = memoize(
   function _getCustomTypeDefinition (customTypeKey) {
-    const getTypeByKeyUri = createRequestBuilder().types
+    const getTypeByKeyUri = createRequestBuilder({
```

Comment:
Yeah, but that did not fully explain why a code in another package affects this...